api/src/main/java/org/reactivestreams/Processor.java

```diff
@@ -12,11 +12,11 @@
 package org.reactivestreams;
 
 /**
- * A Processor represents a processing stage—which is both a {@link Subscriber}
+ * A {@link Processor} represents a processing stage—which is both a {@link Subscriber}
  * and a {@link Publisher} and obeys the contracts of both.
  *
- * @param <T> the type of element signaled to the {@link Subscriber}
- * @param <R> the type of element signaled by the {@link Publisher}
+ * @param <T> the type of element signaled to the {@link Subscriber}.
+ * @param <R> the type of element signaled by the {@link Publisher}.
```

Comment:
Such parameter descriptions are not full sentences themselves so I don't think `.` is needed here and elsewhere.

Reply:
Alright, I'll remove those!